util/testutil/testutil.go

```diff
@@ -342,6 +344,33 @@ func (t *TestData) GetTestCases(c *check.C, in interface{}, out interface{}) {
 	t.output[casesIdx].decodedOut = out
 }
 
+// GetTestCasesT gets the test cases for a test function.
+func (t *TestData) GetTestCasesT(tt *testing.T, in interface{}, out interface{}) {
```

Comment:
Can we reuse the methods in `testkit/testdata`?

Reply:
Yes, I will solve this problem in new PR.